src/components/CardWidget.vue

```diff
@@ -18,6 +18,7 @@
           >
             <slot name="header-right"></slot>
             <b-dropdown
+              v-if="(!!$slots['menu'] && !!$slots['menu'][0])"
```

Comment:
This shouldn't be required now that you've placed the `if` condition on the menu slot itself (which is the right approach):

https://github.com/getumbrel/umbrel-dashboard/blob/3a332361fb28b7aaef7de68aca39c7711796ee8e/src/views/Lightning.vue#L193

Because we wouldn't add empty menu slots anywhere.

Reply:
I would agree with you but if I don't add that then the menu will still be created but it will be empty. This is because there is an if statement on line 17 that renders header-right and the dropdown if either is present. What I can do is break that if statement on line 17 to their own elements?